### ISSUE TITLE
Support TAP14's SubtestDocument

### ISSUE THREAD

**Issue by bowerscd** [python-tap/tappy#122]
[TAP 14](https://testanything.org/tap-version-14-specification.html) supports SubtestDocument, which effectively is the request for #14 - 
currently tappy will impose an error on any documents adhering TAP14 to with this new SubtestDocument:

```shell
 (<file=/tmp/tmpgw_n99ao>, '') Only one plan line is permitted per file.
```

It's trivially workaround-able, however (iterate through failures, remove the one with the exact short description "Only one plan line is permitted per file.").

**Comment by mblayman** [comment]
Yep, my policy was to make tappy compliant with the TAP spec and no more. TAP 14 just got merged (I was one of the contributors), so I haven't invested any time into it yet.

There's a lot of work here to make this work properly. Thanks for submitting the issue.
